nemo-keyboard/org/nemomobile/KeyboardLandscape.qml

```diff
@@ -43,19 +43,25 @@ Column {
     property bool inSymView
     property bool inSymView2
 
-    property variant row1:["q1€", "w2£", "e3$", "r4¥", "t5₹", "y6%", "u7<", "i8>", "o9[", "p0]"]
-    property variant row2: ["a*`", "s#^", "d+|", "f-_", "g=§", "h({", "j)}", "k?¿", "l!¡"]
-    property variant row3: ["z@«", "x~»", "c/\"", "v\\“", "b'”", "n;„", "m:&"]
-    property variant accents_row1: ["", "", "eèéêë", "", "tþ", "yý", "uûùúü", "iîïìí", "oöôòó", ""]
-    property variant accents_row2: ["aäàâáãå", "", "dð", "", "", "", "", "", ""]
-    property variant accents_row3: ["", "", "cç", "", "", "nñ", ""]
+    property variant row1: KLayouts.keyboards[currentKeyboardLayout]["row1"]
+    property variant row2: KLayouts.keyboards[currentKeyboardLayout]["row2"]
+    property variant row3: KLayouts.keyboards[currentKeyboardLayout]["row3"]
+    property variant accents_row1: KLayouts.keyboards[currentKeyboardLayout]["accents_row1"]
+    property variant accents_row2: KLayouts.keyboards[currentKeyboardLayout]["accents_row2"]
+    property variant accents_row3: KLayouts.keyboards[currentKeyboardLayout]["accents_row3"]
 
     property int topPadding: Theme.itemSpacingExtraSmall
     property int bottomPadding: topPadding
     property int leftPadding: Theme.itemSpacingExtraSmall/2
     property int rightPadding: leftPadding
     property int keyHeight: keyArea.height / 4
 
+    property int totalCharButtons: Math.max(row1.length, row2.length, row3.length)
+
+    property int keyHeight: keyArea.height / 4
```

Comment:
This is twice^ 
https://github.com/nemomobile-ux/plugins/pull/3/files#diff-e18e251268b908d6445ca078de61905dR57